src/auth/Auth.js

```diff
@@ -0,0 +1,118 @@
+/* eslint-disable no-return-await */
+/* eslint no-return-await: "error" */
+
+import { appenddata } from '../redux/fitMeUpSlice';
+import { appendspecial } from '../redux/specilatiySlice';
+
+export const signupUser = async (data) => (
+  await (await fetch('http://127.0.0.1:3001/api/v1/users', {
```

Comment:
- [ ] It seems your server needs to run at 3001 port, instead of the default which would be 3000.   Kindly add those instructions on your backend readme:

rails server -p 3001